app/assets/images/payment_icons/fps.svg

```diff
@@ -0,0 +1 @@
1xiQQshzXmPI77Y3h4PNun76xZlxzdpD2biIiIdBYKWCIiEmvYhFerbGNjLwuhF06vLKle Fuhp+B7AHlsDl3d3t2qgu0N3gx7ge+J0B7oBewI9gIqdJ+wKWPl+wTwBqx7YhPM28A7OJjfbZPhb YBvdcpsIttGw9YGw3szWR27rsuTWR9j6dFSxfuolR79djuvqiTc9dgDZ7GAijnM4Fudj4N2aH4O8 H3z/GLdPwMoBc4EngZmR88SMH54+X3sjERER6cwUsEREpM3U1M2t3NyU696U82qo6E4qW2VZegF4 yrpEwbsCeBRVuwfDrMp8a4Bw9z3dLTKjEugOQLA9zELK3Ssw6/Huj+kJpHcIWN2BymYqA0AVWyMb Ad42J+QPD9v/JeCwyYzGd/+KO+vfe4E1bwQ2bfs+d3sHC1k32wJej1OP2xYj1AezLVGwBofNHnlj ytkUgjd6KmyKcmzxdOWmxsZNG/rtvXLD6NranNYkGHLjtANzpAY5uUGOHYlxLM4+eUNiaQeslWAz gZnmPNmlIf2M7l8lIiIi8n4KWCIiIlKyjrr9mYquDVsGEMIRjg8ybBDOIHg3fBZ8o4LSClgOb5n7 LGCWOc+kzGY+9oNPvqaRFhEREYmngCUiIiLtbmBmbmWfXqs/kvX0YRgDCAzA/FDwQ9n+7DnY4QrM kg5Y63CexWyWWZhlpJ594nunLtRoi4iIiBRPAUtERETazJBbp+2TC6mD3e1gcz/UsEMdHwgcBKSb vz1Z/khVIgHLgSXA8wHmgM9xC88+9d3TF2vERURERFpGWotAREREWsqQu6Z1yW5KH5ByO9jND/LA QWAHAweBH5zLbb3f2NZ/QbMC72QJJPiKNrYJeBHneXOez7nPoVvTnJmXDd+g0RcRERFpPToDS0RE RBI5/oYnuqa65Pql3fp7iPq5+UFAf5x+4P2BfsCBQLTte7zAO0Xu8IdmPhR3nV6rnoG1HnyRO/MM 5rqzKAo27/H6v79EJhO0NoiIiIi0LZ2BJSIi0kkdf8MTXaOK0KvSKnt5lOsX3PsDvSK8l7v1I6K/ Qy8L9NoaqLzaQ7T1bRqtrc6MMlr5LKz1ZsxzZ65j89yZm07nFj12+WmLtIaIiIiIlA4FLBERkTIw 5NZpPSKv7E1F1Jus7Z1LWW9zegO9zentxl546A18AKMPzt5AFydFIAf887Rsx97rRrbtE96Rl441 Yr4IbB4wH/eXIveXunnT/Ee+e/omrT0iIiIipU8BS0REWlVNXV1qQ9OAkyOsv7u/Mun8gc9gyU/f GXbzq1VUbznOnUODsZ+brzVsaQhhaVU6WvLw+YevL5fltG71B6utW7RnLpfrRS6qJrJqD6HaYE/M qjGrxkM12J64VwN7YbYX7r2BqgBbr9mLwLa7+7hvK1PbQpSX5aq2GmORBxZHkS+CaBGERSEXFu97 wJvLR9fW5rQ1ioiIiHRcugeWiIi0mmH3zDvOCXcBh2730vOyu99rxiSj6uWJF3z4fTe/HlI3t0dF fTg0ZQx141SCnwRbb/yd5/ZGG8BXAmsN3gz4m4a9SfA3PWK9O43mvtGNzZg3mPGWhbTnQnaLka7f /uGyTY0buveozDVt9uqQMoMGAKJQ2Y10toqmrV8XoIKU9dj+e0MIvSIzI1DteBc3umJU41ZlHrpj 0R7uXuVYz4hcD8eqgD3N6erQA+i54xNr/h3w8t+3ybf9N9k75+38MB5Tttzz/dh3P+gx37Y798B6 7y/1wGKMRZZjMdgiYFEWX1RVFRZP/8bQjdriRERERMqXApaIiLSKT9/zwkiwvwCVO36umWjyDpBl 682/92Tnz+/8fe99IH8c8SLCSfMPF1t7Yr90hzuN7/ARj/m+f36wkwWstcDrDssMlrv76xC9htki j1j0+KUnrCzmzD0RERERKS8KWCIi0uJO/+8Xj7DI/krwAzGqdvx84miyw+cVsJp9sB2/q+QCluOr DF43eD3Aa+b2uuPLCSyLovB6U65i+YwrPr5FW46IiIiI5KOAJSIirepTf36+r1nFflEq7BsCBxi+ rxPt584BZr4v7vsBXfI+gAJW849bGgFrHdgb7qw2wgqc1Wa2MsCKyH1pLu2v98y+8/rEy4Y3aEsQ ERERkd2hgCUiIu1u2J3P9rHK9D4hRP1DxD5R8P4hYh/D+hG8H3g/h/5A1/d9owJWc9+1uwFrI/gK YLVbtAr3lQarPYQVFvnqkEutShGtWLOu5+q5mYGNWntFREREpC0oYImISIdx1h/n99zSZcs+FtJ9 nLC3hWhvPHyAyPu4295m9CHQF6Mvzt6ON39mV+cKWFvYen+p1eCrzX0tZmtx1rr7KvNobc7CmnTa V71N1cpZlxy9WWuaiIiIiJQaBSwRESlbQ+rm9ujamK3OBaoDXh051bjv6VDtUB29+3+g2qA6uPXC vdqgyqE7eDfY7h5e7R+w6sHfMXjLA2uIbC34Wg++2szWAGvB15qHteR8VXqPHmsf+eIRm7QmiIiI iEhHp4AlIiJSwLB7ntwj1PdM56LN1Z6tSKcs19MtdPEo1dVD6OFQEZlVE+JfV3Pv/Sns/IIcbP3W P/jGCJpC8EYs2hjl/C0P4e29+vZ4Z3StLtkTERERERERERERERERERERERERERERERERERERERER ERERERERERERERERERERERERERERERERERERERERERERERERERERERERERERERERERERERERERER EREREREREREREfmn/w+BS718bzxb3wAAACV0RVh0ZGF0ZTpjcmVhdGUAMjAxOC0xMS0wOFQwMzoy MTo0OCswMDowML9tozIAAAAldEVYdGRhdGU6bW9kaWZ5ADIwMTgtMTEtMDhUMDM6MjE6NDgrMDA6 MDDOMBuOAAAAAElFTkSuQmCC" id="image844" opacity="1" stop-opacity="1"/></g></svg>
```

Comment:
1. The `<title id="pi-fps">{FPS}</title>` should be `<title id="pi-fps">FPS</title>`
2. Please remove the `</g>` at the end. It is causing an error to render the svg.